src/CSharp/CodeCracker/Usage/DisposableVariableNotDisposedCodeFixProvider.cs

```diff
@@ -62,6 +62,38 @@ public static SyntaxNode CreateUsing(SyntaxNode root, ObjectCreationExpressionSy
                 var statement = (LocalDeclarationStatementSyntax)variableDeclaration.Parent;
                 newRoot = CreateRootWithUsing(root, statement, u => u.WithDeclaration(variableDeclaration));
             }
+            else if (objectCreation.Parent.IsKind(SyntaxKind.Argument))
+            {
+                var identifierName = GetIdentifierName(objectCreation, semanticModel);
+
+                var whiteSpace = SyntaxFactory.Whitespace(" ");
```

Comment:
No. But I don't think you need it. Just add `.WithAdditionalAnnotations(Formatter.Annotation)`.